misk/src/main/kotlin/misk/web/concurrencylimits/ConcurrencyLimiterStrategy.kt

```diff
@@ -0,0 +1,10 @@
+package misk.web.concurrencylimits
+
+enum class ConcurrencyLimiterStrategy {
```

Comment:
Could some docs be added on what this strategy is and potentially where to learn more about the algos?

Reply:
👍🏼👍🏼👍🏼👍🏼 this is really a first pass to validate the approach, but will follow up with tests and docs!

Reply:
Thanks for these thorough docs!